packages/tools/src/stateManagement/segmentation/config/segmentationColor.ts

```diff
@@ -96,7 +96,18 @@ function getColorForSegmentIndex(
 
   // get colorLUT
   const colorLUT = SegmentationState.getColorLUT(colorLUTIndex);
-  return colorLUT[segmentIndex];
+  let colorValue = colorLUT[segmentIndex];
+  if (!colorValue) {
+    if (
+      segmentIndex < 0 ||
+      segmentIndex > 65535 ||
+      Math.floor(segmentIndex) !== segmentIndex
```

Comment:
what is Math.floor of an integer (segment index)? can you please add a comment what is happening in the code?

Reply:
remove entrierly